eng/config/PublishData.json

```diff
@@ -226,6 +226,19 @@
       "insertionTitlePrefix": "[d17.2p2]"
     },
     "main": {
+      "nugetKind": [
+        "Shipping",
+        "NonShipping"
+      ],
+      "version": "4.2.*",
+      "packageFeeds": "default",
+      "channels": [],
+      "vsBranch": "main",
+      "vsMajorVersion": 17,
+      "insertionCreateDraftPR": true,
```

Comment:
@JoeRobich I believe this should be set to `false` right?

Reply:
If we are shipping for vs-deps, then the insertions from main are just for validation and can be draft prs.